src/util/batchInsert.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+import {isNumber, isString, isArray, chunk, flatten} from 'lodash';
+import Promise from '../promise';
+
+export default class BatchInsert {

```

Comment:
Strange, I've been using x2 tabs all along. Does look weird in this PR tho I agree. Changed it now. *(Does not appear updated in the diff view? https://github.com/tgriesser/knex/pull/1392/commits/65af6a8f39b054f2cbf27118349cdc402d2108ac)*